campaignion_bar/theme/campaignion-bar.tpl.php

```diff
@@ -24,35 +24,4 @@
 
 <div id="ae-widepopups">
   <?php print theme('campaignion_bar_menu', array('widebar' => TRUE, 'links' => menu_tree_all_data('ae-menu'), 'wide' => array('New'))); ?>
```

Comment:
This is still `ae-menu`?

Reply:
Yes, I didn’t touch the menu name (or the styles).